lib/src/main/java/tech/relaycorp/relaydroid/endpoint/AuthorizationBundle.kt

```diff
@@ -0,0 +1,7 @@
+package tech.relaycorp.relaydroid.endpoint
+
+public class AuthorizationBundle(
+    public val pdaSerialized: ByteArray,
+    public val pdaChainSerialized: List<ByteArray>
+) {
+}
```

Comment:
This class will have a `serialize(): ByteArray` method to generate the files we discussed last week on Slack. Behind the scenes, it'll be using a separate class from the core lib.